curations/maven/mavencentral/org.webjars.bowergithub.webcomponents/shadycss.yaml

```diff
@@ -0,0 +1,15 @@
+coordinates:
+  name: shadycss
+  namespace: org.webjars.bowergithub.webcomponents
+  provider: mavencentral
+  type: maven
+revisions:
+  1.5.0-1:
+    described:
+      sourceLocation:
+        name: shadycss
+        namespace: org.webjars.bowergithub.webcomponents
+        provider: github
+        revision: v1.5.0
```

Comment:
Use the commit ID instead of the tag since tags can be changed or deleted.

Reply:
Done (in new PR).